src/quo2/components/wallet/wallet_overview/view.cljs

```diff
@@ -107,7 +107,7 @@
                  :margin-right 8}}
        (time-frame time-frames)]
       (when (and (= state :default) (not= metrics :none))
-        [view-metrics metrics currency-change percentage-change])])])
+        [view-metrics metrics currency-change percentage-change theme])])])
```

Comment:
Consider using a map here instead

Reply:
Thanks for your review @J-Son89!

done